base/rules/best-practices.js

```diff
@@ -215,6 +215,22 @@ module.exports = {
     // http://eslint.org/docs/rules/vars-on-top
     'vars-on-top': 2,
 
```

Comment:
I mean I think we should leave this. We shouldn't encounter this often though since we prefer `const` and `let`.